lib/ae_mdw/dex.ex

```diff
@@ -41,6 +43,7 @@ defmodule AeMdw.Dex do
   @typep page_cursor :: Collection.pagination_cursor()
   @typep query() :: map()
   @typep range :: {:gen, Range.t()} | nil
+  @typep pubkey() :: Db.pubkey()
```

Comment:
Do we gain something from having aliases internal to the modules, instead of using the type directly? For me it is a bit confusing, as I need to check the definition at the top of the module, instead of being certain we are using a global definition and not one specific to this module.

Reply:
In this case it's doesn't change much, but it does help when the type signature is too long. This way it won't break it into multiple lines